packages/smooth_app/lib/pages/onboarding/consent_analytics_page.dart

```diff
@@ -115,60 +77,57 @@ class ConsentAnalytics extends StatelessWidget {
     );
   }
 
+  BottomAppBar _buildBottomAppBar(
```

Comment:
You should use the dedicated Widget for that purpose in Flutter: [ButtonBar](https://api.flutter.dev/flutter/material/ButtonBar-class.html)

Reply:
✅